content/en/doc-con-content.md

```diff
@@ -0,0 +1,171 @@
+     </td>
+    </tr>
+  </table>
+
+- **Avoid repetition.** Repetition is a "mental stumbling block" that gets in the way of the reader's comprehension of the thought or topic. For example:
+
+  <table>
+    <tr>
+     <td>
+  <strong>Do</strong>
+     </td>
+     <td><strong> Don't</strong>
+     </td>
+    </tr>
+    <tr>
+     <td>Smurfcaptor supports single- and multi-Smurf capture events, including batch processing.
+     </td>
+     <td>Smurfcaptor supports single- and multi-Smurf capture events. Smurfcaptor also supports batch processing.
+     </td>
+    </tr>
+  </table>
+
+- **Never use "we."** The reader will wonder whether they're part of the "we" or not. For example:
+
+  <table>
+    <tr>
+     <td>
+  <strong>Do</strong>
+     </td>
+     <td><strong> Don't</strong>
+     </td>
+    </tr>
+    <tr>
+     <td>Version 3.0 adds support for Smurf counting.
+     </td>
+     <td>In version 3.0 we added Smurf counting support.
+     </td>
+    </tr>
+  </table>
+
+- **Never use "might" or "may." **"May" is often confused with "might" but they are not the same. "Might" introduces uncertainty just like "should," while "may" implies the permission has been granted and not that an option or action is available to the user. For example:
+
+  <table>
+    <tr>
+     <td>
+  <strong>Do</strong>
+     </td>
+     <td><strong> Don't</strong>
+     </td>
+    </tr>
+    <tr>
+     <td>The build starts when you click <strong>Build</strong>.
+     </td>
+     <td>The build might start when you click <strong>Build.</strong>
+     </td>
+    </tr>
+    <tr>
+     <td>You can choose either high or low Smurf sensitivity.
+     </td>
+     <td>You may choose high or low Smurf sensitivity.
+     </td>
+    </tr>
+  </table>
+
+## Calling out important things
+
+To call out something of particular importance, use the three severity levels of callouts: note, caution, and warning.
+<table>
+  <tr>
+   <td><strong>Callout Type</strong>
+   </td>
+   <td><strong>Example</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Note</strong> calls the reader's attention to something important.
+   </td>
+   <td><strong>Note:</strong> It is best to capture Smurfs in batches, as processing a large number of Smurfs at once can result in long wait times.
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Caution</strong> indicates something that may cost the reader additional time and work if not followed. 
+   </td>
+   <td><strong>Caution:</strong> Papa Smurf is a highly skilled mage. To maximize your Smurf yields, avoid capturing him along with other Smurfs.
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Warning</strong> warns the reader of a potential catastrophic failure.
+   </td>
+   <td><strong>Warning: </strong>Looking directly into the Basilisk's eyes turns you instantly to stone.
+   </td>
+  </tr>
+</table>
+
+## Things to avoid
+
+Avoid the following:
+- **Copy-pasting large portions of existing documentation.** If you need to reuse a large piece of content,

```

Comment:
```suggestion

- **Copy-pasting large portions of existing documentation.** If you need to reuse a large piece of content,
```
need a blank line before start of a list to ensure proper formatting

Reply:
Fixed.